src/operator/bufferToggle.ts

```diff
@@ -44,18 +46,18 @@ import {errorObject} from '../util/errorObject';
  * @owner Observable
  */
 export function bufferToggle<T, O>(openings: Observable<O>,
-                                   closingSelector: (value: O) => Observable<any>): Observable<T[]> {
+                                   closingSelector: (value: O) => SubscribableOrPromise<any> | void): Observable<T[]> {
```

Comment:
/cc @david-driscoll - I remember we already have couple of signature followed this pattern. May need to update them as well to clear.